### ISSUE TITLE
using `channel_events` with Redis results in worker crash

### ISSUE THREAD

**Issue by yashka713** [slact/nchan#480]
I'm trying to use `channel_events` and this is my config:
```
location ~ /pub/(\w+)$ {
                nchan_publisher;
                nchan_channel_id                $1;
                nchan_use_redis                 on;
                nchan_message_timeout           1m;
               nchan_channel_events_channel_id $1;
}

location ~ /channel_events/(.+) {
                nchan_subscriber;
                nchan_channel_group meta;
                nchan_channel_id $1;
}
```
And in the nginx `error.log` log I see:
```
2018/08/13 10:41:26 [alert] 10032#10032: worker process 10041 exited on signal 11 (core dumped)
ker process: /build/nginx/debian/modules/nchan/src/util/nchan_msg.c:151: msg_release: Assertion `msg->refcount > 0' failed.
2018/08/13 10:41:40 [alert] 10032#10032: worker process 10723 exited on signal 6 (core dumped)
ker process: /build/nginx/debian/modules/nchan/src/util/nchan_msg.c:120: msg_release: Assertion `msg->storage != NCHAN_MSG_SHARED' failed.
```
server:
CPU: 1*1.8
RAM: 1GB

nchan version: 1.2.1

@slact, maybe, it's not enought?

**Comment by slact** [comment]
fixed in master with commit b10148a8375cbd16a1c , with tests for meta channel events added in ec360933fd6f29f0e. Had I added that test earlier, I would have caught this 1-line fix much earlier.

**Comment by slact** [comment]
Gonna close this one.
